cmd/fleetctl/session_test.go

```diff
@@ -27,8 +29,7 @@ func TestEarlySessionCheck(t *testing.T) {
     token: phIEGWGzKxXui1uZYFBXFwZ1Wv1iMxl79gbqMbOmMxgyZP2O5jga5qyhvEjzlGsdM7ax93iDqjnVSu9Fi8q1/w==`
 	err := ioutil.WriteFile(configPath, []byte(config), configFilePerms)
 	require.NoError(t, err)
-	_, exitErr, err := runAppNoChecks([]string{"get", "queries", "--config", configPath})
-	require.Error(t, err)
-	require.NotNil(t, exitErr)
-	require.True(t, errors.Is(err, invalidSessionErr))
+
+	_, err = runAppNoChecks([]string{"get", "queries", "--config", configPath})
+	require.ErrorIs(t, err, service.ErrUnauthenticated)
```

Comment:
Nice! TIL about `ErrorIs`! (I was used to using `require.True(t, errors.Is(err, service.ErrUnauthenticated))`)